app/uk/gov/hmrc/nisp/views/service_error_500.scala.html

```diff
@@ -0,0 +1,42 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.nisp.views.html.main
+@import uk.gov.hmrc.nisp.utils.Constants
+@import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
+@import play.api.Application
```

Comment:
What is this used for?

Reply:
removed, as isn't required. Copied as a template and forgot to remove unused imports.